.appveyor.yml

```diff
@@ -29,6 +29,10 @@ environment:
           PYTHON_ARCH: 64
           MSVC_VERSION: "Visual Studio 10 Win64"
 
+        - PYTHON_VERSION: 2.7
+          PYTHON_ARCH: 64
+          MSVC_VERSION: "Visual Studio 10 Win64"
```

Comment:
what motivated this change? if we want to support Python 2.7 on Windows we need to use older compilers and enforce C89 standard in our C source files I think. Maybe more trouble than it is worth though.

Reply:
I was just playing spot the difference between the two builds and hoping it was a typo.  I didn't realise we had fundamental problems like that.